website/content/userguide/minio.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "setting up MinIO on gokrazy"
+weight: 50
+---
+
+# Setting up MinIO on gokrazy
+
+Since MinIO is a storage solution, you need to enable permanent storage on your
+gokrazy installation by running the `mkfs` command that `gokr-packer` prints. For
+more details, see [Quickstart](/quickstart/).
+
+This article also assumes that you have an instance directory (with a `go.mod`)
+set up.
+
+## Step 1: Configuring the environment variables and command-line flags
+
+You can find a detailed description on how to set the flags and environment vars
+in the article [per-package configuration](/userguide/package-config/).
+
+`${INSTANCE?}/env/github.com/minio/minio/env.txt`
+```env
+MINIO_ROOT_USER=minio
+MINIO_ROOT_PASSWORD=minio-on-gokrazy
+```
+
+`${INSTANCE?}/flags/github.com/minio/minio/flags.txt`
+```
+server
+--address
+:3001
+--console-address
+:3002
+/perm/minio/
+```
+
+A few things can be noted here:
+
+* There are nice examples on the different (advanced) options in the minio docs
```

Comment:
Can you add a link to the minio docs you’re referring to?

Also, please change minio to MinIO for consistency.